test/byContract/IexecPocoBoost/IexecPocoBoostDelegate.test.ts

```diff
@@ -1170,7 +1170,7 @@ describe('IexecPocoBoostDelegate', function () {
             ).to.be.revertedWith('PocoBoost: Overpriced workerpool');
         });
 
-        it('Should fail when the workerpool tag does not provide what app, dataset and request expect', async function () {
+        it('Should fail when workerpool tag does not match all app, dataset and request tags', async function () {
```

Comment:
Minor but previous wording is more accurate (we can update though, something like "Should fail when the workerpool tag does not provide what app provider, dataset provider and requester want")

Reply:
Updated, LMK.